setup.py

```diff
@@ -10,8 +10,7 @@
 from typing import Optional  # NOQA
```

Comment:
I guess you can remove `# NOQA` comments.

Reply:
Okay